cpp/order/SolidLiquid.cc

```diff
@@ -29,7 +29,9 @@ void SolidLiquid::compute(const freud::locality::NeighborList* nlist,
 
     // Compute Steinhardt using neighbor list (also gets ql for normalization)
     m_steinhardt.compute(&m_nlist, points, qargs);
-    const auto& qlm = m_steinhardt.getQlm();
+    const auto& qlm = m_steinhardt.getQlm()[0];
+    // Even though this is a 2-D array since Steinhardt can compute multiple l the second dimension is always
+    // 1 since we only allow one l for SolidLiquid.
```

Comment:
Put the comment above the line to which it applies. Also this comment can be much simpler.
```suggestion
    // SolidLiquid only has one l value so we index the 2D array from Steinhardt.
    const auto& qlm = m_steinhardt.getQlm()[0];
```